### ISSUE TITLE
Use aria-pressed on the various toolbars buttons in a consistent way

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#1878]
The [aria-pressed](https://www.w3.org/TR/wai-aria/states_and_properties#aria-pressed) attribute is an ARIA state to be used for buttons that "require a full press-and-release cycle to change their value. Activating it once changes the value to true, and activating it another time changes the value back to false."

Within the editor, many toolbars buttons indicate the state of the block content (alignments, formatting, etc.): for example on some bold text it's perfectly correct to have the Bold button in an `aria-pressed="true"` state so that communicates to assistive technologies it is pressed, and matches its visual "pressed" look.

However, `aria-pressed` does change also the way assistive technologies announce the buttons. Normal buttons are announced just as `button` while buttons with an `aria-pressed` attribute are announced as `toggle button`"

![screen shot 2017-07-13 at 09 52 49](https://user-images.githubusercontent.com/1682452/28158769-18f3ab46-67bb-11e7-8e71-b03e3ab54757.png)
![screen shot 2017-07-13 at 09 52 52](https://user-images.githubusercontent.com/1682452/28158772-198bd0ec-67bb-11e7-994b-a35afb77f0d7.png)

Note: different assistive technologies may use a different wording, for example NVDA announces (maybe more correctly?):
`Align right toggle button not pressed`
`Align right toggle button pressed`

For this reason, it's important to set an initial `aria-pressed` state on these buttons. Some of them have an initial state, for example the ones used for the Text Block, but other buttons in other toolbars miss the initial state.

In the "Classic Text" block, for example:

- all alignments buttons: no `aria-pressed`
- quote: no `aria-pressed`
- unordered, ordered list: they do have `aria-pressed`
- bold, italic, strikethrough: no `aria-pressed`

Also, the more button shouldn't have an `aria-pressed` attribute; instead, it should indicate it has a submenu and its state using:
  - aria-haspopup="true" 
  - aria-expanded

<img width="560" alt="screen shot 2017-07-13 at 09 10 31 3" src="https://user-images.githubusercontent.com/1682452/28159625-d9a4cbac-67bd-11e7-8f12-c1cad7222ea5.png">

**Comment by mtias** [comment]
I believe this is consistent now looking at quote, paragraph, etc. There is a difference in Classic where the role is applied to the wrapper div element, though.

**Comment by afercia** [comment]
Yep I think the Gutenberg toolbar buttons use `aria-pressed` consistently. Classic Text is still inconsistent though, `aria-pressed` should be applied to the DIV with `role="button"`. @mtias should this be reported upstream?

**Comment by Afraithe** [comment]
@mtias yes, we are looking into it.

**Comment by mtias** [comment]
Thanks!
